README.md

```diff
@@ -226,13 +226,15 @@ necessary:
    `windows`.
 3. map the container's port on localhost 4000 to the hosts port 4000
 
-With our `dev_device` from above, the call would then look, for example, as follows:
+With our `dev_device` from above, the call on a Windows host would then look,
+for example, as follows:
 
 ```sh
 docker run --rm \
   -u 0:0 \
   -v "~/.ssh":/root/.local/share/omnect-cli \
   -v dev_env.toml:/dev_env.toml \
+  -e CONTAINER_HOST=windows \
```

Comment:
missing: 
-e CONTAINERIZED=true

Reply:
This variable is pre-set on docker build (see the Dockerfile) doesn't need to be set explicitly. I mentioned in the README for sake of completeness.